shamir/shamir.go

```diff
@@ -127,16 +127,16 @@ func div(a, b uint8) uint8 {
 // mult multiplies two numbers in GF(2^8)
 // GF(2^8) multiplication using log/exp tables
 func mult(a, b uint8) (out uint8) {
-	var goodVal, zero uint8
-	log_a := logTable[a]
-	log_b := logTable[b]
-	sum := (int(log_a) + int(log_b)) % 255
+	var zero uint8
```

Comment:
Let's back out all changes in this file except those relating to renaming variables.

Reply:
✅